test/prism_regression/float.rb

```diff
@@ -0,0 +1,3 @@
+# typed: true
```

Comment:
Maybe you can add some `*.parse-tree.exp` files to ensure the produced AST is correct?

Like what's done with https://github.com/sorbet/sorbet/blob/master/test/testdata/parser/kwnil.parse-tree.exp.

Reply:
Good idea! I'll look into this now.